src/components/Input/index.js

```diff
@@ -113,12 +113,12 @@ export default class Input extends PureComponent {
               ref={this.input}
             />
 
-            {label &&
+            {(error || label) &&
               <label
                 htmlFor={name}
                 className='mc-form-input__label'
               >
-                {label}
+                {label} {error}
```

Comment:
```suggestion
                {error || label}
```

Reply:
I, initially, had exactly that.  Errors that come from the server are actually incredibly useful in the format they are in:

```json
{ "email": "can't be blank" }
```

leads well to be rendered like `Email can't be blank`

Reply:
My idea fails if the label isn't the name of the field.

`Tell us how you would describe yourself can't be blank` fails...  Changing.  Thanks!